### DISCUSSION TITLE
Server test says unable to write, file manager does nothing

### DISCUSSION THREAD

**Discussion by counterpoint** [GetPublii/Publii#1508] [Help]
I am trying to use Publii with an existing HTML web site. A "Test Connection" on the server page shows an error saying "able to connect but unable to store files". However, I can send files to the server using scp with the same parameters as are entered into Publii. 

File Manager says there are no files, but I am not clear where it is looking. "Sync your web site" gives an error about connecting to the server. I don't know where Publii is trying to sync to. 

Where should I look for solutions?

**Comment by dziudek** [comment]
@counterpoint - most probably the "Remote path" option is incorrectly set. If you are using SFTP, please make sure that you are using absolute path to your destination catalog in this option.

**Reply by counterpoint** [reply]
What is the catalog? I was hoping Publii might work as a remote editor for a web site that contains only a lot of HTML files. Maybe it isn't so simple!

**Reply by counterpoint** [reply]
The remote path in the server page is definitely correct for pointing to the web site. I can use scp to send files to that path.
